### PR TITLE
Create BNFabb122.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New record.

### CODE REVIEW

File: ParisBNF/abb/BNFabb122.xml [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+         <particDesc>
+           <listPerson>
+               <person>
+                 <persName ref="PRS1071dAbbadi" role="owner"/>
+              </person>
+              <person>
+                 <persName ref="PRS1071dAbbadi" role="bequeather"/>
+                 </person>
+                   <person>
+                 <persName ref="PRS13770EntonyosWaldaAbadia" role="owner"/>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb122.xml:342]: again PRS1071dAbbadi?

File: ParisBNF/abb/BNFabb122.xml [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+                            <textLang mainLang="gez"/>
+                            </msItem>
+
+
+                    <msItem xml:id="ms_i2">
+                          <locus from="64ra" to="68rb"/>
+                          <title type="complete" ref="LIT3177Canon"/>
+                          <textLang mainLang="gez"/>
+                          <incipit xml:lang="gez">
+                          <hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩አምላክ።</hi> ቀኖዳ፡ ዘአንበሩ፡
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb122.xml:49]: ቀኖና?

File: ParisBNF/abb/BNFabb122.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb122.xml:342]: Sorry, I do not understand. Why again?

Comment (eu-genia) [reply ParisBNF/abb/BNFabb122.xml:342]: https://github.com/BetaMasaheft/Persons/pull/534#pullrequestreview-904178717

File: ParisBNF/abb/BNFabb122.xml [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+                                <msItem xml:id="ms_i4">
+                                      <locus from="149ra" to="168ra"/>
+                                      <title type="complete" ref="LIT6574AssartuSelat"/>
+                                      <textLang mainLang="gez"/>
+                                      <incipit xml:lang="gez">
+                                        <hi rend="rubric">በስመ፡ እግዚአብሔር፡ ኃያል፡</hi> ንወጥን፡ በረድኤተ፡ እግዚ<hi rend="rubric">አብሔር፡
+                                        ሎቱ፡ ስብሐት፡ በጸ</hi>ሒፈ፡ ዓሠርቱ፡ ተስእለታ<hi rend="rubric">ት፡ ተስአሎ፡ ቀዳማዊት፡ ዘ</hi>
+                                        ብዩን፡ ውስቴታ፡ ትርጓሜ፡ ትሥልስት፡ ወተዋሕዶ፨ ይቤ፡ ረድእ፡ ኦመምህር፡ ነገር፡ ኅፁር፡ ብያኔ፡ ለዓለማውያን፡
+                                      </incipit>
+                                      <note>published by Cerulli 1960 (Scritti teologici etiopici dei secoli XVI-XVII, II: La Storia dei Quattro Concili ed altri opuscoli
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb122.xml:74]: Could you mark up the references?

File: ParisBNF/abb/BNFabb122.xml [hunk 4]
```diff
@@ -0,0 +1,369 @@
...
+                                                ሊቃውን<hi rend="rubric">ተ፡ ቤተ፡ ክርስቲያን፡ ርቱዓነ፡ ሃይማኖት፡ ትብል፡ ከ</hi>መዝ፨
+                                                ነአምን፡ ወንትአምን፡ በልብ፡ ወበቃል፡ በአብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡
+                                              </incipit>
+                                                </msItem>
+
+
+                                          <msItem xml:id="ms_i7">
+                                          <locus from="193rb" to="213ra"/>
+                                            <title type="complete" ref="LIT3560Chrfaith"/>
+                                            <textLang mainLang="gez"/>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb122.xml:105]: am

File: ParisBNF/abb/BNFabb122.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb122.xml:342]: If you mean that Entonyos can be Antoin, yes, it is possible, but not proved yet, I made a remark in the PRS13770EntonyosWaldaAbadia, should I say smth in Abb 122?

**Review by thea-m — CHANGES_REQUESTED** [review]

File: ParisBNF/abb/BNFabb122.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb122.xml:74]: ok

File: ParisBNF/abb/BNFabb122.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb122.xml:49]: ok

**Review by eu-genia — APPROVED** [review]

**Review by thea-m — APPROVED** [review]

**Pull request merged**